internal/handlers/pg/msg_createindexes.go

```diff
@@ -273,7 +273,6 @@ func processIndexKey(keyDoc *types.Document) (pgdb.IndexKey, error) {
 		var orderParam int64
 
 		if orderParam, err = commonparams.GetWholeNumberParam(order); err != nil {
-			// TODO Add better validation and return proper error: https://github.com/FerretDB/FerretDB/issues/2311
```

Comment:
I removed this TODO and the TODO below because `ErrNotImplemented` seems to be the best value we can return now. For some special index types, order can be set differently than 1 or -1, but we don't support those index types yet.